TerminalDocs/command-line-arguments.md

```diff
@@ -4,33 +4,142 @@ description: Learn how to create command line arguments for Windows Terminal.
 author: cinnamon-msft
 ms.author: cinnamon
 ms.date: 05/19/2020
-ms.topic: overview
+ms.topic: how-to
 ms.service: terminal
 ---
 
-# Create Command Line Arguments for Windows Terminal
+# Create command line arguments for Windows Terminal
 
-Instructions on how to create Command Line Arguments for Windows Terminal.
+You can use `wt.exe` to open a new instance of Windows Terminal from the command line. You can also use the execution alias `wt` instead. If you built the Windows Terminal from the source code on [GitHub](https://github.com/microsoft/terminal), you can open that build using `wtd.exe` or `wtd`. This will always open a new window. Windows Terminal is not yet capable of opening new tabs or panes to an existing instance.
```

Comment:
```suggestion
You can use `wt.exe` to open a new instance of Windows Terminal from the command line. You can also use the execution alias `wt` instead. If you built the Windows Terminal from the source code on [GitHub](https://github.com/microsoft/terminal), you can open that build using `wtd.exe` or `wtd`. This will always open a new window. Windows Terminal is not yet capable of opening new tabs or panes in an existing instance.
```

"to" -> "in" near the end